examples/loader-datawrapper/src/index.md

```diff
@@ -0,0 +1,83 @@
+# Datawrapper data loader
+
+Here’s a Python data loader that generates an HTML [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) tag that embeds a [Datawrapper](https://www.datawrapper.de/) chart.
+
+```python
+import sys
+
+import pandas as pd
+from datawrapper import Datawrapper
+
+# Get an extract of arrests made by the Baltimore Police Department
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/palewire/first-automated-chart/main/_notebooks/arrests.csv",
+    parse_dates=["ArrestDateTime"]
+)
+
+# Create a year column
+df['year'] = df.ArrestDateTime.dt.year
+
+# Calculate the total number by year
+totals_by_year = df.year.value_counts().sort_index().reset_index()
+
+# Connect to the datawrapper api
+dw = Datawrapper()
+
+# Create a chart
+chart_config = dw.create_chart(
+    # Headline the chart
+    title="Baltimore Arrests",
+    # Set the type
+    chart_type="column-chart",
+    # Give the data
+    data=totals_by_year,
+    # Configure other bits
+    metadata={
+        # Set the description
+        "describe": {
+            "source-name": "OpenBaltimore",
+            "source-url": "https://data.baltimorecity.gov/datasets/baltimore::bpd-arrests/about",
+        }
+    }
+)
+
+# Pull out the chart's unique identifier
+chart_id = chart_config["id"]
+
+# Publish the chart
+dw.publish_chart(chart_id)
+
+# Write the chart's embed code to stdout
+html = dw.get_iframe_code(chart_id, responsive=True)
+sys.stdout.write(html)
+```
+
+<div class="note">
+
+To run this data loader, you will need to create an API token in Datawrapper and set it as an environment variable named `DATAWRAPPER_ACCESS_TOKEN`.
```

Comment:
It would be nice to have detailed instructions to create the venv. In particular since this repo expects the venv to live in .venv (judging from the .gitignore file).

I tried this:

```sh
python -m venv .venv
source .venv/bin/activate
pip3 install -r requirements.txt
```

but it failed with:
```
error: externally-managed-environment

× This environment is externally managed
```

To make this work I had to run the last command with the `--break-system-packages` option 😅:
```sh
python -m venv .venv
source .venv/bin/activate
pip3 install -r requirements.txt --break-system-packages
```

(Maybe it's just me—I appear to be doomed when I need to install python stuff.)

Reply:
I'm agnostic on Python package managers, so I'm open to handling this however you want.

My routine is to use pipenv. So, in my world, I run something like this:

```bash
pipenv install datawrapper pandas
pipenv run npm run dev
```

I put the requirements file in my proposals because I saw it that way in other examples and presumed it was your preference. It's not the canonical way to run pipenv, [but it's supported](https://docs.pipenv.org/en/latest/advanced.html#importing-from-requirements-txt), so here's how I did things in this case.

```bash
pipenv shell
pipenv install -r requirements.txt
```

I don't have any experience with venv but there must be a way to make it work without resorting to hacks or unusual options. Let me know how you'd like to proceed.

Reply:
wow, I didn't know about `pipenv`; maybe it can save me from my personal python hell!